src/TreeSitter/Parser.hs

```diff
@@ -28,7 +28,7 @@ withParser language action = Exc.bracket
   ts_parser_new
   ts_parser_delete
   $ \ parser -> do
-    ts_parser_set_language parser language
+    _ <- ts_parser_set_language parser language
```

Comment:
We should probably actually change the signature of this function to return in `Maybe` based on whether we get `True` or `False` back from this function, but that doesn’t necessarily happen in this PR; feel free to file an issue and punt 👍